app/controllers/auth.rb

```diff
@@ -8,11 +8,18 @@ class SlowFoodApp
   end
 
   get '/signup' do
-
+    erb :signup
   end
 
   get '/users' do
     @users = User.all
     erb :users_index
   end
+
+  post '/register' do
+    user_params = params['user']
+    user = User.new(user_params)
+    user.save
+    redirect '/', notice: "Thank you for signing up #{user.name}"
```

Comment:
So what would happen if the `user` object is not able to be saved?

Reply:
We will create the if statement. First we're trying to enable gem flash to work properly for the first message

Reply:
Right.

Reply:
Solved the issue, moving on to implement the sad path